### PR TITLE
Enable foundation wasm tests in pipeline

### PR DESCRIPTION
**Pull request by Schahen**
Enable foundation wasm tests in pipeline - gradually we'll introduce more and more.

### CODE REVIEW

File: .github/workflows/wasm-pr.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'zulu'
+      - name: install Android SDK
+        run: ./jbdeps/android-sdk/downloadAndroidSdk
+      - name: Run Wasm tests
+        run: ./gradlew :compose:foundation:foundation:wasmJsBrowserTest --info
```
Comment (eymar) [inline .github/workflows/wasm-pr.yml]: Jfyi: we have a combined test task - :mpp:testWeb 
It currently tests only runtime on js. We can add more dependencies to it and call here only a combined task.

Not sure if the agents have Chrome installed by default or recent versiont. If Chrome installation is required then you can have a look at Skiko github workflow for web.

Comment (Schahen) [reply .github/workflows/wasm-pr.yml]: @eymar pushed relevant changes

File: .github/workflows/wasm-pr.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Skiko Unit tests
```
Comment (eymar) [inline .github/workflows/wasm-pr.yml]: I guess we will run only web targets in this workflow, right?

Maybe let's name more spefic. Something like:

"Web Unit Tests"?

Comment (eymar) [reply .github/workflows/wasm-pr.yml]: Hmm. Where does it take "iOS Unit tests results" here?

Screenshot 2024-01-23 at 16 00 39

**Comment by eymar** [comment]
The test is failing because of timeout (doesn't fail locally though, but CI agent is likely less powerful).

We have two options IMO: 
- Use less iterations here - https://github.com/JetBrains/compose-multiplatform-core/blob/90bf6742f79af8f31ba560c299cb133e1cccf46f/compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/text/TextFieldFocusTest.kt#L74
- or add
```
config.client.mocha = config.client.mocha || {};
config.client.mocha.timeout = 60000;
```
to `../compose-multiplatform-core/mpp/karma.config.d/wasm/config.js`

I like 2nd option more. @Schahen what do you think?

**Review by eymar — APPROVED** [review]

**Pull request merged**